packages/react-native-web/src/exports/StatusBar/index.js

```diff
@@ -7,16 +7,92 @@
  * @providesModule StatusBar
  * @flow
  */
-
 import { Component } from 'react';
 
+
+const {head} = document
+
+let _barStyle = 'default'
+let _hidden = false
+let _translucent = false
+
+
+function getMetaTag(attrName) {
+  const htmlCollection = head.getElementsByTagName('meta')
+
+  let tag = Array.from(htmlCollection).filter(({name}) => name === attrName)[0]
+
+  if(!tag) {
+    tag = document.createElement('meta')
+    tag.name = attrName
+
+    head.appendChild(tag)
+  }
+
+  return tag
+}
+
+function setAppleMobileWebAppCapable() {
+  getMetaTag('apple-mobile-web-app-capable').content =
+    (_hidden || _translucent || _barStyle !== 'default') ? 'yes' : 'no'
+}
+
+function setAppleMobileWebAppStatusBarStyle() {
+  setAppleMobileWebAppCapable()
+
+  getMetaTag('apple-mobile-web-app-status-bar-style').content =
+    _translucent ? 'black-translucent' : _barStyle
+}
+
+
 export default class StatusBar extends Component<*> {
-  static setBackgroundColor() {}
-  static setBarStyle() {}
-  static setHidden() {}
+  static get currentHeight() {
+    const {availHeight, height} = screen
+
+    return height - availHeight
+  }
+
+
+  static setBackgroundColor(color, animated) {
+    getMetaTag('theme-color').content = color
+  }
+
+  static setBarStyle(style, animated) {
+    if(style === 'light-content') style = 'black'
+
+    if(!['black', 'black-translucent', 'default'].includes(style)) return
```

Comment:
`includes` is not polyfilled and it's not necessary to create an array to perform this check

Reply:
I'm used to use `includes()`, but it's ok for me to change that. Done.